tensorflow/tools/ci_build/install/install_deb_packages.sh

```diff
@@ -23,6 +23,10 @@ apt-get install -y \
     build-essential \
     curl \
     git \
+    gfortran \
```

Comment:
We need scipy inside virtualenv, for the install+test. This file is for outside virtualenv. Without these new packages, pip install scipy and pip install sklearn will fail in the virutalenv.